spec/lib/tasks/partners_rake_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+require 'rake'
+
+describe 'partners rake tasks' do
+  before do
+    Rake.application.rake_require 'tasks/partners'
+    Rake::Task.define_task(:environment)
+  end
+
+  describe 'partners:seed_users' do
+    let!(:prev_csv_file) { ENV['CSV_FILE'] }
+    let!(:prev_email_domain) { ENV['EMAIL_DOMAIN'] }
+
+    after do
+      ENV['CSV_FILE'] = prev_csv_file
+      ENV['EMAIL_DOMAIN'] = prev_email_domain
+      Rake.application['partners:seed_users'].reenable
+    end
+
+    context 'with missing CSV_FILE' do
+      before do
+        ENV.delete('CSV_FILE')
+        ENV['EMAIL_DOMAIN'] = 'foo.com'
+      end
+
+      it 'displays an error message' do
+        expect { Rake::Task['partners:seed_users'].invoke }.to \
+          output("You must define the environment variables CSV_FILE and EMAIL_DOMAIN\n").to_stdout
+      rescue SystemExit
```

Comment:
maybe we could add an `around` block to the group instead of adding this to each spec?

```
around do |ex|
  ex.run
rescue SystemExit
end
```

Reply:
great tip, thanks!